README.md

```diff
@@ -49,3 +54,30 @@ with connect(...) as conn:
 It also implements the `close()` method, as suggested by the PEP-2049
 specification, to support situations where the cursor is wrapped in a
 `contextmanager.closing()`.
+
+### Runtime and region selection
+
+You can chose the Wherobots runtime you want to use using the `runtime`
+parameter, passing in one of the `Runtime` enum values. For more
+information on runtime sizing and selection, please consult the
+[Wherobots product documentation](https://docs.wherobots.com).
+
+The only supported Wherobots compute region for now is `aws-us-west-2`,
+in AWS's Oregon (`us-west-2`) region.
+
+### Advanced parameters
+
+The `connect()` method takes some additional parameters that advanced
+users may find useful:
+
+* `results_format`: one of the `ResultsFormat` enum values;
+    Arrow encoding is the default and most efficient format for
```

Comment:
You might want to rephrase the defaults here as well.

Reply:
Comment is still accurate about the system's behavior, which is why I didn't change them.